HSLdevcom/jore-map-ui
# 639 Add entityTypeToggles in routeview

To test:

- open route view
- search for routes to add
- line/node toggle should be visible and functional

Closes #639
